packages/mds-api-authorizer/index.ts

```diff
@@ -14,11 +14,41 @@ const {
   TOKEN_USER_EMAIL_CLAIM = 'https://ladot.io/user_email'
 } = process.env
 
-export type ApiAuthorizer = (req: express.Request) => ApiAuthorizerClaims | null
+export type ApiAuthorizer = (req: express.Request | string) => ApiAuthorizerClaims | null
```

Comment:
These are pretty heterogeneous types, it seems odd to pass in a string. I guess if it does take a string it just returns `null`? This probably deserves a comment explaining what's happening.